cypress/e2e/challenges.cy.js

```diff
@@ -88,4 +89,13 @@ describe('Challenge Tests', () => {
     cy.dataCy(ChallengesPage.SUCCESS_ALERT).should('contain', 'Your answer is correct!')
     cy.dataCy(ChallengesPage.PROGRESS_BAR).should('be.visible').should('not.have.attr', 'aria-valuenow', '0')
   })
+
+  it('Submitting right answer gives visual cue on homepage that the challenge is successfully solved', () => {
+    cy.visit('/challenge/0')
+    cy.dataCy(ChallengesPage.ANSWER_TEXTBOX).type('The first answer')
+    cy.dataCy(ChallengesPage.SUBMIT_TEXTBOX_BTN).click()
+    cy.dataCy(ChallengesPage.SUCCESS_ALERT).should('contain', 'Your answer is correct!')
```

Comment:
I think this could be removed from this test. If it fails at this point it may be a little unclear why. I think if the assertion on line 99 is great though!